### PR TITLE
Added AutoFish module.

### PR DESCRIPTION
**Pull request by ghost**
I could also use a non-sound method to make this work, but is it really worth it? I believe by using the sound packets it should be just fine, as long as you don't go extremely far.

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAutoFish.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                        player.swingHand(Hand.MAIN_HAND)
+                    }
+                }
+
+                if (player.offHandStack.item is FishingRodItem) {
+                    repeat(2) {
+                        network.sendPacket(PlayerInteractItemC2SPacket(Hand.OFF_HAND))
+                        player.swingHand(Hand.OFF_HAND)
+                    }
+                }
```
Comment (superblaubeere27) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAutoFish.kt]: This can be replaced by a loop.

**Review by superblaubeere27 — CHANGES_REQUESTED** [review]
Thank you for implementing this. I think that there could be more settings and there should be an option that automatically redeploys the bobber. Additionally Auto Fish should check if the player is has actually deployed the bobber since there could be some problems with the modules making players using their rod seemingly randomly

**Comment by ghost** [comment]
> Thank you for implementing this. I think that there could be more settings and there should be an option that automatically redeploys the bobber. Additionally Auto Fish should check if the player is has actually deployed the bobber since there could be some problems with the modules making players using their rod seemingly randomly

These two suggestions are added. The only problem left is to replace the `main/off`HandStack.item checks to loops, but unfortunately I can't think of a way to make it as a loop.

**Comment by superblaubeere27** [comment]
> 
> 
> > Thank you for implementing this. I think that there could be more settings and there should be an option that automatically redeploys the bobber. Additionally Auto Fish should check if the player is has actually deployed the bobber since there could be some problems with the modules making players using their rod seemingly randomly
> 
> These two suggestions are added. The only problem left is to replace the `main/off`HandStack.item checks to loops, but unfortunately I can't think of a way to make it as a loop.

Uh you could do this: `
for (hand in arrayOf(Hand.MAIN_HAND, Hand.OFF_HAND)) {
     ...
}`

I will polish the code and merge it.

**Comment by ghost** [comment]
Didn't think of that but alright.

**Review by superblaubeere27 — APPROVED** [review]
👍🏻

**Comment by ghost** [comment]
Yep, should've known that method.

**Pull request merged**